index.html

```diff
@@ -252,22 +252,21 @@
                             <div class="floe-front-news">
                                 <h3>News</h3>
 
+                                <h4>Self-Assessment Learning Toolkit</h4>
+                                <p class="floe-date">August 23, 2017</p>
+                                <p>We've been working on developing an inclusive self-assessment toolkit that allows learners to become experimental researchers in the subject of their own learning. Recently, the design team has been exploring fun new ways of engaging the learner in collecting data.</p>
+                                <p><a href="./news/2017-08-23-self-assessment-toolkit.html">Read more</a></p>
```

Comment:
There are three "Read more" links for each of the news articles. @dayotte could you do us a favour and make those "Read more" links say something more descriptive? Maybe:

Read (or Read more?): "Self-Assessment Learning Toolkit"
Read: "Inclusive Science Lab"
Read: "DEEP Conference 2016"

Reply:
@jhung you mean the visible link text, right? not a label or otherwise?